src/App.vue

```diff
@@ -1,16 +1,29 @@
 <template>
   <div id="app">
-    <Animals />  
+    <AnimalCreation @animalCreated="updateList" />
+    <Animals :key="animalListKey" />
```

Comment:
A quoi sert animalListKey , ou tu l'utilises ?

Reply:
`animalListKey` sert de clé d'identification de la vue `Animals`, c'est la manière la plus propre que j'ai trouvé pour mettre à jour la liste des animaux après un ajout à celle-ci.

Source : https://michaelnthiessen.com/force-re-render/ -- conseillé dans plusieurs topics pour le refresh de listes

Reply:
Il dis ca a la fin de son post :
==>
Just remember, if you find yourself needing to force Vue to re-render a component, maybe you aren't doing something the best way.